awx/main/models/jobs.py

```diff
@@ -274,6 +274,15 @@ class Meta:
             'admin_role',
         ],
     )
+    prevent_instance_group_fallback = models.BooleanField(
```

Comment:
maybe you need to add it to the list of FIELDS_TO_PRESERVE_AT_COPY or something like that. Minor detail, but best to wrap it up now.